.changeset/sour-ties-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/java': major
+---
+
+Replace java reserved keyword new with \_new
```

Comment:
Do we really need major release again?

Reply:
@ardatan no. I've issued another changeset in commit https://github.com/dotansimha/graphql-code-generator/pull/6012/commits/302fd3b322c048d1976a2a982a7b4ea8ee684523 ... 
Did it not catch it? :(

Reply:
I think you need to rebase your branch because this branch seems to add `major` changeset again which has been already released :)

Reply:
rebase? but it's my branch, i've done the commit and push.
can we re-run the PR build? If it fails to identify the changeset, I don't mind requesting a new PR and close this one

Reply:
Actually, come to think of it, this is a major change. I'm restoring the "old" way of creating the constructor with args map.

Reply:
@ardatan ? can you please merge with major?

Reply:
@ardatan thank you for the quick response! :)